service/connection/QueryRunner.kt

```diff
@@ -74,7 +74,7 @@ class QueryRunner constructor(
 
             class ConceptMapGroups : Stream<ConceptMapGroup>()
             class NumericGroups : Stream<NumericGroup>()
-            class ConceptMaps constructor(val source: Source) : Stream<ConceptMap>() {
+            class ConceptMapsWithQuery constructor(val source: Source, val query: TypeQLQuery) : Stream<ConceptMap>() {
```

Comment:
We store the relevant query with the concept map stream here. This isn't ideal, but we'd need to make larger refactors to how we consume responses in order to reach something more canonical.